lib/mongo/server/connection.rb

```diff
@@ -167,6 +167,7 @@ def ping
       def timeout
         @timeout ||= options[:socket_timeout]
       end
+      alias :socket_timeout :timeout
```

Comment:
why `alias` instead of rename the method? is it used somewhere else? or just safety in case someone calls it from outside?